apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/SnifferConfigInitializer.java

```diff
@@ -40,7 +42,7 @@
  * The <code>SnifferConfigInitializer</code> initializes all configs in several way.
  */
 public class SnifferConfigInitializer {
-    private static final ILog logger = LogManager.getLogger(SnifferConfigInitializer.class);
+    private static ILog LOGGER = LogManager.getLogger(SnifferConfigInitializer.class);
```

Comment:
Why change this?

Reply:
The name `logger` cannot pass style-check. Maybe something wrong with the style-check preference?

Reply:
It used to be able to pass, then it still can. All codes in the master branch have to pass the style check. Please recheck again.

In the rule, we set an exception for the logger due to the legacy issue.

Reply:
@lujiajing1126 , what kind of style check do you mean, we don't check that, did you use a plugin in the IDE that's not relevant to SkyWalking's code style?

Reply:
> @lujiajing1126 , if you have a `final`, it can be lowercase `logger`, why remove `final`?

Yes, exactly. It cannot pass without `final`. 

I remove `final` here because I want to re-create the `logger` instance with the updated logger resolver set up just after the config initialization.  On line 105 the logger will possibly be called again.

Reply:
Then, I have a feeling, this PR should be after the all `logger` and the exception rule to be removed. I don't want the codes with 2 style logger.

Reply:
> Then, I have a feeling, this PR should be after the all `logger` and the exception rule to be removed. I don't want the codes with 2 style logger.

Agree. I have reverted the styles, just wait for the polish of the style-check rules.

Reply:
Would you like to do that first? I may not have time to do this recently. You could remove the old exception rule, once style check passed, you have changed all of them.

Reply:
> Would you like to do that first? I may not have time to do this recently. You could remove the old exception rule, once style check passed, you have changed all of them.

Sure. I can do this. Do you prefer a new PR with only rule changes?

Reply:
Rule change and code adoption, otherwise, the CI will fail.